### ISSUE TITLE
'Flutter release notes' page issue

### ISSUE THREAD

**Issue by domesticmouse** [flutter/website#4121]
Page URL: https://flutter.dev/docs/development/tools/sdk/release-notes/
Page source: https://github.com/flutter/website/tree/master/src/docs/development/tools/sdk/release-notes/index.md

Description of issue: Release notes for Flutter 1.17.1 is MIA.

**Comment by sfshaza2** [comment]
Yes, the 1.17.0 should be relabeled to either 1.17 or 1.17.1.

**Comment by domesticmouse** [comment]
I suspect linking the 1.17 Medium release post isn't the right thing here. All the other links here are to the release notes with list of bugs and features, and we should be doing the same for 1.17.x

**Comment by miguelcmedeiros** [comment]
I agree with the last comment from @domesticmouse. Opening a medium article when the user actually wants to look at the release notes is not straightforward. I had to re-read the article to find the [link to flutter.dev](https://flutter.dev/docs/development/tools/sdk/release-notes/changelogs/changelog-1.17.0). 

Aside from that, the release notes in flutter.dev for 1.17 are less useful compared to the previous releases. I found the way it was structured with a summary, breaking changes, new features, etc. For 1.17 it's a dump of all PRs integrated per repo.

For 1.20 I couldn't find the release notes page. Is it intentional?

Although I find the medium articles a nice reading describing what's new in the release, it's not really what I expect to see when I visit flutter.dev looking for release notes. 
I also understand that it's a lot of work putting together release notes for a project like Flutter given the crazy number of PRs that get merge between releases, but as a user, it's actually quite useful.

I hope this is something that can be addressed in the upcoming releases.

**Comment by sfshaza2** [comment]
This was an intentional change as of 1.17. (I didn't even realize it at first.) 

I'm wondering if this issue might get more traction if filed on the flutter/flutter site.

@kf6gpe, can you weigh in?

**Comment by kf6gpe** [comment]
Hi! Thanks for the feedback! We've been experimenting with a number of things in how we do release notes, as you've seen, so getting your thoughts is great.

* We had (perhaps) incorrectly assumed that having both a news-style blog post and the change log was perhaps too much information for most of our users, and was an additional chunk of work to assemble, which is why we stepped away from it starting in 1.17 and moved to a dump of PRs, and then stopped doing it entirely for 1.20, because you can get the same thing with a GitHub query at https://github.com/flutter/flutter/issues. I'm sorry we've removed something you like!

* We shifted to posting release notes on Medium in the Flutter channel, and cross-linking. I think it's worth having a conversation with the community as to whether that's the right idea or not. The general thinking is that it's news as much as a Flutter update, and we can engage more prospective users on Medium. Again, open to input!

* We've never been in the habit of writing release notes for x.y.z releases outside of the first one for an x.y release, although occasionally our stable release _was_ a dot release. We've made some changes in our release process, and now can guarantee that the first x.y release to stable will always have a trailing '0'  --- i.e., 1.20.0. This should remove the confusion where "1.17.x" was the first stable release, yet got release notes as if it were 1.17.0.

@Sfshaza I'm going to move this to flutter/flutter, as I think you're right.

@csells and @timsneath Let's discuss this more as we plan for our next stable release in light of this and any other feedback we get.
